test/fuzzTesting/fuzzHelper.cpp

```diff
@@ -84,9 +84,17 @@ std::string RandomCode::generate(int _maxOpNumber, RandomCodeOptions _options)
 			}
 		}
 		else
+		{
+			if (info.name.find_first_of("PUSH") > 0)
```

Comment:
If I am not mistaken, this searches the string info.name for any of the letters 'P', 'U', 'S' and 'H'. Is that the intention?